websites/docs-smoke-test/src/content/components/video-player.mdx

```diff
@@ -0,0 +1,20 @@
+---
+title: Video component
+wideLayout: true
+---
+
+# Video Player
+
+## Video 1
+<Video
+  url="https://customer-ytbpo1yna9xohg5m.cloudflarestream.com/e1aa2e24614b242afdd4afb36a0c2f75/manifest/video.m3u8"
+  posterPath="https://customer-ytbpo1yna9xohg5m.cloudflarestream.com/e1aa2e24614b242afdd4afb36a0c2f75/thumbnails/thumbnail.jpg?time=6s"
```

Comment:
if it takes URLs or paths dynamically the propname would better be just "poster" (back to the original).  a URL is definitely not a "path"